files/galaxy/tpv/destinations.yml

```diff
@@ -101,17 +100,10 @@ destinations:
       dependency_resolution: "none"
       submit_request_cpus: "{cores}"
       submit_request_memory: "{mem}"
-      # $defaults expands to "$galaxy_root:ro,$tool_directory:ro,$job_directory:ro,$working_directory:rw,$default_file_path:rw"
-      docker_volumes: "$defaults,/cvmfs/data.galaxyproject.org:ro"
-      singularity_volumes: "$defaults,/cvmfs/data.galaxyproject.org:ro"
+      docker_volumes: "$default,$job_directory:rw,$tool_directory:ro,{{ cvmfs.data.path }}:{{ cvmfs.data.docker_perm }}"
     scheduling:
       require:
         - pulsar
-      accept:
-        - conda
-        - singularity
-        - docker
-        - condor-tpv
```

Comment:
We may consider merging changes affecting scheduling tags in a different PR (or a least a specific commit just for this) to make things clearer. Of course at this point that involves interactive rebase and force push :wink: